tests/postgres_tests/test_array.py

```diff
@@ -307,6 +307,28 @@ def test_adding_field_with_default(self):
         with connection.cursor() as cursor:
             self.assertNotIn(table_name, connection.introspection.table_names(cursor))
 
+    @override_settings(MIGRATION_MODULES={
+        "postgres_tests": "postgres_tests.array_index_migrations",
+    })
+    def test_adding_arrayfield_with_index(self):
+        table_name = 'postgres_tests_chartextarrayindexmodel'
+        call_command('migrate', 'postgres_tests', verbosity=0)
+        with connection.cursor() as cursor:
+            conslike = [
```

Comment:
conslike -> like_constraint_field_names?
also, assertions could be moved out of the "with" statements.